osu.Game.Rulesets.Osu/Edit/TransformToolboxGroup.cs

```diff
@@ -51,9 +51,17 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            // aggregate two values into canRotate
+            RotationHandler.CanRotatePlayfieldOrigin.BindValueChanged(_ => updateCanRotateAggregate());
+            RotationHandler.CanRotateSelectionOrigin.BindValueChanged(_ => updateCanRotateAggregate());
```

Comment:
This is going against standard bindable practices of avoiding binding events to a bindable owned by something else. I think you need to make local bindables for these.

Reply:
So a local bindable bound to the external bindable, and then add the event to that local bindable? 

Seems clunky to me but sure.

Reply:
It's he only way we can ensure that events aren't left bound beyond a component's lifetime. I'll make the change.

Reply:
Now you say it out, it makes sense. Thanks for the heads up